sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueServiceVersion.java

```diff
@@ -62,7 +62,12 @@
     /**
      * Service version {@code 2021-04-10}.
      */
-    V2021_04_10("2021-04-10");
+    V2021_04_10("2021-04-10"),
+
+    /**
+     * Service version {@code 2021-04-10}.
+     */
+    V2021_06_08("2021-06-08");
```

Comment:
Does this need to be the `getLatest` value?

Reply:
See other comment on the same thing